accounts/views.py

```diff
@@ -25,30 +26,27 @@ def post(self, request):
             'password': password
         }
         response = requests.post(
-            'http://yashasingh.tech:8085/api/profiles/login/', data=data)
+            'http://yashasingh.tech:8085/api/profiles/login/',
```

Comment:
Remove this string and import the endpoints from another file. It is easy to manage and a good practice. Never use API endpoints directly. :slightly_smiling_face: